File: picture_load.py
# Load a picture from a file.

import harfang as hg

pic = hg.Picture()

if hg.LoadPicture(pic, "resources/pictures/owl.jpg"):
	print("Picture dimensions: %rx%r" % (pic.GetWidth(), pic.GetHeight()))
else:
	print("Failed to load picture!")

File: picture_load.go
package main

import (
	"fmt"

	hg "github.com/harfang3d/harfang-go/v3"
)

func main() {
	pic := hg.NewPicture()

	if hg.LoadPicture(pic, "resources/pictures/owl.jpg") {
		fmt.Printf("Picture dimensions: %dx%d", pic.GetWidth(), pic.GetHeight())
	} else {
		fmt.Println("Failed to load picture!")
	}
}
